src/System.Management.Automation/resources/RemotingErrorIdStrings.resx

```diff
@@ -952,7 +952,7 @@ All WinRM sessions connected to Windows PowerShell session configurations, such
     <value>A running command could not be found for this PSSession.</value>
   </data>
   <data name="NetFrameWorkV2NotInstalled" xml:space="preserve">
-    <value>The Microsoft .NET Framework 2.0, which is required for Windows PowerShell 2.0, is not installed. Install the .NET Framework 2.0 and retry.</value>
+    <value>The Microsoft .NET Framework 2.0, which is required for PowerShell 2.0, is not installed. Install the .NET Framework 2.0 and retry.</value>
```

Comment:
Windows PS specific, please revert although PS2.0 is deprecated, so maybe this error and code should be removed in another PR

Reply:
Fixed.